### DISCUSSION TITLE
没人抬 | 连波的闲谈杂鱼

### DISCUSSION THREAD

**Discussion by giscus** [godruoyi/gblog#31] [Announcements]
# 没人抬 | 连波的闲谈杂鱼

乡里人怕的，是自己死了后，找不到人抬，我不明白。

https://godruoyi.com/posts/the-village/

**Comment by sangmingming** [comment]
我们老家以前是这样的，不过这几年都外出打工了，大家也都看的越来越明白了，不是亲戚，不是过年人在家的喜事，也很多不去了。

**Reply by godruoyi** [reply]
是啊，所以现在村里过年都没之前热闹了。哎，人啊，一方面感叹村里的规矩太过迂腐，一方面又怀念儿时村里的风土人情。你说，这是不是一种矛盾呢？

**Reply by scopewu** [reply]
这是那种离土思乡，却又没办法长时间呆下去。

你们那儿的风景不错，大山风景还得云贵地区。我们那儿也是山区，但是树木没有这么茂盛，云雾缭绕。

博客主题不错。

**Reply by godruoyi** [reply]
谢谢你的回复，人真实一种奇怪的动物，故乡也真是一个神奇的地方啊

**Comment by scopewu** [comment]
“白事”关系亲近的确实应该回去，其他看个人。我们那儿还没有到这种地步，不过确实是靠乡里人帮忙，近些年席饭多是包工，轻松不少。

**Reply by godruoyi** [reply]
谢谢你的回复，白事我觉得回去倒是能接受，我有点反感的是无论大小事，村里人的规矩都是都要回来；或许是我们常年在外，等我们再老十年说不定就能理解了，

**Comment by GoodBoyboy666** [comment]
其实如果真正关系好的，哪怕当事人不叫我我也会回去帮忙，如果自己实在有事去不了，当事人也不会过多的计较，这种就称之为“情”。那种建立在利益上的塑料关系，才被称为“人情世故”。和金钱一样，都是可抛弃，可重新拥有的，如果自己觉得没有必要，完全可以抛弃掉。而“乡里人”则是介于两者之间，自己真正是否需要只有自己知道，如果对自己没有任何帮助，为何不直接降本增效，什么抬棺都是伪需求，只是很多人看不清罢了。
